doc/Learn/Maui/MauiOverview.md

```diff
@@ -49,6 +51,28 @@ dotnet new unoapp -preset blank -maui -o SimpleMauiApp
 
 .NET MAUI Embedding can be added to any existing Uno application via the following steps. The .NET MAUI Embedding feature is only supported in Uno applications that target iOS, Android, MacCatalyst, and Windows, as shown in the Platforms folder in the following solution structure. These instructions are appropriate for an Uno application targeting .NET 7.
 
+### MauiEmbedding Dependencies
+
+# [**Single Project**](#tab/single-project)
+
+![Starting solution structure - SingleProject](Assets/GettingStartedExisting01-StartSingleProjectStructure.png)
+
```

Comment:
Just to confirm, no need to add the [Uno.Extensions.Maui.WinUI](https://www.nuget.org/packages/Uno.Extensions.Maui.WinUI) reference?

Reply:
AFAIK all packages are brought automatically when we add `MauiEmbedding` to `UnoFeatures`. Can you confirm @jeromelaban ?